app/assets/stylesheets/application.css

```diff
@@ -25,3 +21,361 @@ a {
   text-decoration: none;
   color: black;
 }
+
+/* category.css */
```

Comment:
- [ ] Please make sure that all tests run and pass successfully. Currently, there are failing tests evident by the following screenshot.

![image](https://github.com/mershark/budget-app/assets/62290545/fb499fe9-78c8-4730-ad4e-d8c28cbfbba6)